lnbits/bolt11.py

```diff
@@ -1,366 +1,5 @@
-import hashlib
-import re
-import time
-from decimal import Decimal
-from typing import List, NamedTuple, Optional
+from bolt11.decode import decode  # noqa: F401
+from bolt11.encode import encode  # noqa: F401
+from bolt11.types import Bolt11
```

Comment:
Manual assignment can be replaced by that:
```suggestion
from bolt11.types import Bolt11 as Invoice
```